experiments/LES/dycoms.jl

```diff
@@ -140,7 +140,7 @@ function config_dycoms(FT, N, resolution, xmax, ymax, zmax)
                                      solver_type=CLIMA.ExplicitSolverType(LSRK144NiegemannDiehlBusch),
                                      ref_state=ref_state,
                                      C_smag=C_smag,
-                                     moisture=EquilMoist(5),
+                                     moisture=EquilMoist(),
```

Comment:
Are these changes needed?

Reply:
This will break dycoms, please revert this.

Reply:
No I'll clean that up.